handlers/bounty.go

```diff
@@ -229,6 +229,18 @@ func (h *bountyHandler) CreateOrEditBounty(w http.ResponseWriter, r *http.Reques
 		bounty.Tribe = "None"
 	}
 
+	// if bounty.PhaseUuid == nil {
+	// 	w.WriteHeader(http.StatusBadRequest)
+	// 	json.NewEncoder(w).Encode("PhaseUuid is a required field")
```

Comment:
@Vayras Please remove this commented lines of code.

Reply:
@elraphty sure thing